infrastructure/batch/compute_environment.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_batch_compute_environment" "scpca_portal" {
```

Comment:
I think we will want to change `scpca_portal` here to something more specific. I do believe at this time we will want to allocate something smaller for ad hoc downloads in the future as there wont really be a benefit to throwing more vcpus at that. Perhaps something like `scpca_portal_project_compute` since this would be for generating downloadable files at the project level.

Reply:
I updated this one, and the name of the ecs_task role. I think the rest are okay for now. My feeling is that we can always adjust things later on, but for now things look consistent.